modules/bim/lib/open_project/bim/bcf_xml/exporter.rb

```diff
@@ -127,8 +127,8 @@ def topic_markup_file(issue_dir, issue)
     def viewpoints_for(issue_dir, issue)
       [].tap do |files|
         issue.viewpoints.find_each do |vp|
-          vp_file = File.join(issue_dir, vp.viewpoint_name)
-          snapshot_file = File.join(issue_dir, vp.snapshot.filename)
+          vp_file = File.join(issue_dir, "#{vp.uuid}.xml")
+          snapshot_file = File.join(issue_dir, "#{vp.uuid}#{vp.snapshot.extension}")
```

Comment:
@oliverguenther Doesn't that mean that if you import BCF XML with viewpoint files that have a different name than the UUID that we will lose that name on the next export? So the export would not be identical to the import. Maybe I am remembering it wrong but I think we tried to keep it the same.

Reply:
You're correct. But I think in this case it's safe to rename the viewpoints. I've seen most examples do that in the BCF-XML test cases

Reply:
you mean test cases of other applications? 

OK, for us it is important that the export works. If at some later point people tell us a good reason why we should do that differently, we can still change it.